firmware/ci.c

```diff
@@ -27,6 +27,18 @@
 
 int status_enabled;
 
+static const struct {
```

Comment:
I feel like this structure might belong in `edid.h` or `edid.c`? Is it here because it is only needed for parsing/string generation?